sdk/js/src/relayer/__tests__/wormhole_relayer.ts

```diff
@@ -0,0 +1,690 @@
+import { afterAll, beforeEach, describe, expect, jest, test} from "@jest/globals";
+import { PublicKey } from "@solana/web3.js";
+import { ethers } from "ethers";
+import { generateRandomString, waitForRelay } from "./utils/utils";
+import {getAddressInfo} from "../consts" 
+import {getDefaultProvider} from "../main/helpers"
+import {
+    relayer,
+    ethers_contracts,
+    tryNativeToUint8Array,
+    ChainId,
+    CONTRACTS,
+    CHAIN_ID_TO_NAME
+  } from "../../../";
+  import {GovernanceEmitter, MockGuardians} from "../../../src/mock";
+
+  const env = process.env['ENV'];
+  if(!env) throw Error("No env specified: tilt or ci or testnet or mainnet");
+  const network = env == 'tilt' || env == 'ci' ? "DEVNET" : env == 'testnet' ? "TESTNET" : env == 'mainnet' ? "MAINNET" : undefined;
+  if(!network) throw Error(`Invalid env specified: ${env}`);
+
+  const sourceChainId = network == 'DEVNET' ? 2 : 6;
+  const targetChainId = network == 'DEVNET' ? 4 : 14;
+
+// Devnet Private Key
+const privateKey = process.env['WALLET_KEY'] || "4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
```

Comment:
what's that smell...? *sniffs*

It's the smell of ... code duplication!

Reply:
Removed :)